A few more suggestions (#13)
janaiyengar/ack-frequency: draft-iyengar-quic-delayed-ack.md

As specified in Section 13.3.1 of {{QUIC-TRANSPORT}}, an endpoint SHOULD
immediately acknowledge packets marked with the ECN Congestion Experienced (CE)
codepoint in the IP header, even if the `disable_ack_on_reordering` transport

> parameter is present. Doing so reduces the peer's response time to congestion

Reviewer: I don't think this is necessary. The TP says nothing about ECN, and ECN says nothing about reordering.

Author: Removed.

While it is expected that endpoints will have only one ACK-FREQUENCY frame in
flight at any given time, this extension does not prohibit having more than one
in flight. Generally, when using `max_ack_delay` for PTO computations, endpoints

> MUST use the maximum of the current value and all those in flight.  Once all in

Reviewer: ```suggestion
MUST use the maximum of the current value and all those in flight.  If there are
```

> flight ACK-FREQUENCY frames are acknowledged, the `max_ack_delay` from the frame

Reviewer: ```suggestion
ACK-FREQUENCY frames in flight, then endpoints MUST use the `max_ack_delay` value
```

> with the largest Sequence Number is used.

Reviewer: ```suggestion
from the most recent frame (frame with the highest sequence number) that was acknowledged.
```
